File: dotnet/DesignPatterns/CreationalDesignPatterns/FactoryDesignPattern/FactoryDesignPattern.Example1/Services/Concretes/PushNotificationService.cs
using FactoryDesignPattern.NotificationAPI.Services.Abstracts;

namespace FactoryDesignPattern.NotificationAPI.Services.Concretes;

public class PushNotificationService : INotificationService
{
	public string Send(string to, string message)
	{
		string result = $"Sending Push Notification to {to} with message: {message}";
		Console.WriteLine(result);

		return result;
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/FactoryDesignPattern/FactoryDesignPattern.Example1/src/main/java/creational/factory/example1/services/concretes/PushNotificationService.java
package creational.factory.example1.services.concretes;

import creational.factory.example1.services.abstracts.INotificationService;

public class PushNotificationService implements INotificationService {
    public String send(String to, String message) {
        String result = "Sending Push Notification to " + to + " with message: " + message;
        System.out.println(result);
        return result;
    }
}
